packages/one-app-bundler/utils/getConfigOptions.js

```diff
@@ -22,13 +22,17 @@ function validateOptions(options) {
   }
 
   if (options.requiredExternals || options.providedExternals) {
-    const intersection = Object.keys(commonConfig.externals)
-      .filter((externalName) =>
-        // eslint-disable-next-line implicit-arrow-linebreak
-        (options.requiredExternals || options.providedExternals).includes(externalName)
-      );
-    if (intersection.length > 0) {
-      throw new Error(`@americanexpress/one-app-bundler: Attempted to bundle ${intersection.join(', ')}, but modules cannot provide externals that One App includes.`);
+    if (Array.isArray(options.requiredExternals) || Array.isArray(options.providedExternals)) {
```

Comment:
Would there be a case where one of them is an array and the other one is not? The OR operator here suggests that is a valid scenario?

Reply:
There will not be a case that there will be both required and provided externals in the same module. If both are provided then there is an error thrown. This is checking that is any of them are there, then check if its in the array format